docs/rules/standard.md

```diff
@@ -184,24 +184,44 @@ Consistent removal (default) or adding of trailing comma's on call site.
 
+!!! note
+    Trailing comma on call site is automatically disabled if [Wrapping](#wrapping) rule (before 0.48.0, [Indentation](#indentation) rule) is disabled. Because it cannot provide proper formatting with unwrapped calls. (see [Dependencies](./dependencies.md))
+
+    ```kotlin
+    processFoo(Foo(
+        a = 3,
+        b = 4,
+    ),) // it's weird to insert "," between unwrapped (continuated) parenthesis
+    ```
+
 Rule id: `trailing-comma-on-call-site`
 
 ## Trailing comma on declaration site
 
 Consistent removal (default) or adding of trailing comma's on declaration site.
 
 !!! important
-KtLint uses the IntelliJ IDEA `.editorconfig` property `ij_kotlin_allow_trailing_comma` to configure the rule. When this property is enabled, KtLint *enforces* the usage of the trailing comma at declaration site while IntelliJ IDEA default formatter only *allows* to use the trailing comma but leaves it to the developer's discretion to actually use it (or not). KtLint values *consistent* formatting more than a per-situation decision.
+    KtLint uses the IntelliJ IDEA `.editorconfig` property `ij_kotlin_allow_trailing_comma` to configure the rule. When this property is enabled, KtLint *enforces* the usage of the trailing comma at declaration site while IntelliJ IDEA default formatter only *allows* to use the trailing comma but leaves it to the developer's discretion to actually use it (or not). KtLint values *consistent* formatting more than a per-situation decision.
 
 !!! note
-In KtLint 0.48.x the default value for using the trailing comma on declaration site has been changed to `true` except when codestyle `android` is used.
+    In KtLint 0.48.x the default value for using the trailing comma on declaration site has been changed to `true` except when codestyle `android` is used.
 
     The Kotlin coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#names-for-test-methods) encourages the usage of trailing comma's on the declaration site, but leaves it to the developer's discretion to use trailing comma's on the call site. But next to this, it also states that usage of trailing commas has several benefits:
     
      * It makes version-control diffs cleaner – as all the focus is on the changed value.
      * It makes it easy to add and reorder elements – there is no need to add or delete the comma if you manipulate elements.
      * It simplifies code generation, for example, for object initializers. The last element can also have a comma.
 
+!!! note
+    Trailing comma on declaration site is automatically disabled if [Wrapping](#wrapping) rule (before 0.48.0, [Indentation](#indentation) rule) is disabled. Because it cannot provide proper formatting with unwrapped declarations. (see [Dependencies](./dependencies.md))
+
+    ```kotlin
+    class FooWrapper(val foo: Foo = Foo(
+        a = 3,
+        b = 4,
+    ),) // it's weird to insert "," between unwrapped (continuated) parenthesis
+    ```
```

Comment:
What was your reasoning behind adding this particular code block? It might become more clear when you add a comment on line 193 as well.

I have clarified the example with preferred/disallowed code block sample.

Reply:
[Duplicate of above](https://github.com/pinterest/ktlint/pull/1800#discussion_r1091867499)